worker/buildbot_worker/base.py

```diff
@@ -320,6 +320,19 @@ def remote_getWorkerInfo(self):
                 if os.path.isfile(filename):
                     with open(filename, "r") as fin:
                         files[f] = fin.read()
+
+        if os.path.exists('/etc/os-release'):
+            with open('/etc/os-release', "r") as fin:
+                for line in fin:
+                    # as per man page: Lines beginning with "#" shall be ignored as comments.
+                    if line.startswith('#'):
+                        continue
```

Comment:
I think we need a test that covers this line.

Reply:
that might be bit difficult. any ideas how?